Bottom: eedc51bf194c9c590b13d03f965b405cc296f945
Top:    35fe2584499985d9925de2dc86b08c26f1a6aa93
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-01-25 21:48:03 +0800

vhost_net: stop device during reset

Signed-off-by: Jason Wang <jasowang@redhat.com>


---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 7baa90a..8032394 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -1227,6 +1227,7 @@ static long vhost_net_reset_owner(struct vhost_net *n)
 	}
 	vhost_net_stop(n, &tx_sock, &rx_sock);
 	vhost_net_flush(n);
+	vhost_dev_stop(&n->dev);
 	vhost_dev_reset_owner(&n->dev, umem);
 	vhost_net_vq_reset(n);
 done:
